### PR TITLE
Fix #6362 consolidate run-*sh scripts

### LINKED ISSUE #6362: consolidate etc/run*sh scripts to describe alternatives better

### LINKED ISSUE #6375: login_success_response not rendering local uri properly in dev
This is caused by is_react_sim_type not doing the right thing when react isn't running.

### PR DESCRIPTION
**Pull request by robnagler**
-  Fix #6375 feature_config.ui_react controls globally
- run-server.sh and run-supervisor.sh not done; good enough as is.

### CODE REVIEW

File: etc/run.sh [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+    export SIREPO_SMTP_FROM_EMAIL=$USER+support@localhost.localdomainn
+    export SIREPO_SMTP_SEND_DIRECTLY=1
+    export SIREPO_SMTP_SERVER=localhost
+}
+
+_env_moderate() {
+    declare sim_type=$1
+    export SIREPO_FEATURE_CONFIG_MODERATED_SIM_TYPES=$sim_type
+    export SIREPO_AUTH_ROLE_MODERATION_MODERATOR_EMAIL=$USER+moderator@localhost.localdomain
+    _msg "Moderated sym_type=$sim_type"
```
Comment (rorour) [inline etc/run.sh:31]: sym->sim?

File: etc/run.sh [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+    export SIREPO_AUTH_METHODS=guest:basic
+    export SIREPO_FEATURE_CONFIG_API_MODULES=status
+    _msg "To test:
+
+curl -u '$u:$SIREPO_AUTH_BASIC_PASSWORD' http://localhost:8000/server-status
+"
+    _exec_all
+}
+
+_op_test_mail() {
```
Comment (rorour) [inline etc/run.sh:192]: Haven't looked too deep into it but this fails for me locally when ~/mail already exists, and probably unrelated, tests/auth/email1_test.py is failing ci

**Review by rorour — CHANGES_REQUESTED** [review]
Nice change. Maybe we should document the usage of this script somewhere

File: etc/run.sh [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+    # Order is important: jupyterlab-server should be last so it isn't
+    # overwritten with a newer version.
+    declare p=$(pip freeze)
+    declare -a i=()
+    declare f
+    for f in \
+        jupyterhub==1.4.2 \
+        jupyterlab==3.1.14  \
+        notebook==6.5.4 \
+        jupyterlab_server==2.8.2\
```
Comment (e-carlin) [inline etc/run.sh:94]: This should be jupyterlab-server (`-` not `_`)

File: etc/run.sh (hunk 2 above)
Comment (robnagler) [reply etc/run.sh:192]: 7cac2f975. email1_test failure is described in #6383.

File: etc/run.sh (hunk 3 above)
Comment (robnagler) [reply etc/run.sh]: 7cac2f975

File: etc/run.sh (hunk 1 above)
Comment (robnagler) [reply etc/run.sh]: e622a6fe

File: etc/run.sh [hunk 4]
```diff
@@ -0,0 +1,243 @@
+#!/bin/bash
+#source ~/.bashrc
```
[the remark this answers is not in the document]
Comment (robnagler) [reply etc/run.sh]: e622a6fe

**Review by e-carlin — CHANGES_REQUESTED** [review]
Looks good except no_smtp_mail didn't work.
```
sirepo (6362-run-s)$ bash etc/run.sh no_smtp_mail
Oct 04 16:14:27 71750     0 ../sirepo/pkcli/service.py:223:tornado ip=0.0.0.0 port=8000
Traceback (most recent call last):
  File "/home/vagrant/.pyenv/versions/py3/bin/sirepo", line 33, in <module>
    sys.exit(load_entry_point('sirepo', 'console_scripts', 'sirepo')())
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/sirepo_console.py", line 18, in main
    return pkcli.main("sirepo")
  File "/home/vagrant/src/radiasoft/pykern/pykern/pkcli/__init__.py", line 158, in main
    res = argh.dispatch(parser, argv=argv)
  File "/home/vagrant/.pyenv/versions/3.9.15/envs/py3/lib/python3.9/site-packages/argh/dispatching.py", line 183, in dispatch
    for line in lines:
  File "/home/vagrant/.pyenv/versions/3.9.15/envs/py3/lib/python3.9/site-packages/argh/dispatching.py", line 294, in _execute_command
    for line in result:
  File "/home/vagrant/.pyenv/versions/3.9.15/envs/py3/lib/python3.9/site-packages/argh/dispatching.py", line 277, in _call
    result = function(*positional, **keywords)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/pkcli/service.py", line 208, in server
    tornado()
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/pkcli/service.py", line 224, in tornado
    sirepo.modules.import_and_init("sirepo.uri_router").start_tornado(
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/modules.py", line 39, in import_and_init
    _i("sirepo.auth", ["simulation_db"])
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/modules.py", line 24, in _i
    m.init_module(**{k: values[k] for k in kw})
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/auth/__init__.py", line 140, in init_module
    _init_full()
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/auth/__init__.py", line 116, in _init_full
    m = importlib.import_module(pkinspect.module_name_join((p, n)))
  File "/home/vagrant/.pyenv/versions/3.9.15/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/auth/email.py", line 18, in <module>
    import sirepo.smtp
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/smtp.py", line 108, in <module>
    _init()
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/smtp.py", line 83, in _init
    _cfg = pkconfig.init(
  File "/home/vagrant/src/radiasoft/pykern/pykern/pkconfig.py", line 276, in init
    _iter_decls(decls, res)
  File "/home/vagrant/src/radiasoft/pykern/pykern/pkconfig.py", line 733, in _iter_decls
    r[kp] = _resolver(d)(k, d)
  File "/home/vagrant/src/radiasoft/pykern/pykern/pkconfig.py", line 816, in _resolve_value
    return decl.parser(res)
  File "/home/vagrant/src/radiasoft/sirepo/sirepo/smtp.py", line 39, in _cfg_from_email
    pkconfig.raise_error(f"invalid from_email={value}")
  File "/home/vagrant/src/radiasoft/pykern/pykern/pkconfig.py", line 477, in raise_error
    raise AssertionError(msg)
AssertionError: invalid from_email=DevSupport
```

**Comment by robnagler** [comment]
e38f9323 fixes no_smtp_mail

**Review by e-carlin — APPROVED** [review]

**Comment by e-carlin** [comment]
@rorour leaving the final approval and merge to you

**Review by rorour — APPROVED** [review]
Looks good to me

**Pull request merged**